CHANGELOG/CHANGELOG-2.2.md

```diff
@@ -0,0 +1,72 @@
+# Changelog 2.2
+
+## 2.2.0 - 2024-10-30
+
+### Added
+
+- Add a new feature `PacketCapture` to allow users to capture live traffic and upload captured packets to a specified location:
+  - Add PacketCapture API. ([#6257](https://github.com/antrea-io/antrea/pull/6257), [@hangyan])
+  - Add PacketCapture data path support. ([#6756](https://github.com/antrea-io/antrea/pull/6756), [@hangyan])
+  - Refer to [this document](https://github.com/antrea-io/antrea/blob/release-2.2/docs/packetcapture-guide.md) for more information about this feature.
+- Add a few new antctl sub-commands for `BGPPolicy` feature to improve usability:
+  -  `antctl get bgppolicy` to get effective BGP policy applied on the Node. ([#6646](https://github.com/antrea-io/antrea/pull/6646), [@Atish-iaf])
+  -  `antctl get bgppeers` to print current status of all BGP peers of effective BGP policy applied on the local Node. ([#6689](https://github.com/antrea-io/antrea/pull/6689) [#6755](https://github.com/antrea-io/antrea/pull/6755), [@Atish-iaf])
+  -  `antctl get bgproutes` to print the advertised BGP routes. ([#6734](https://github.com/antrea-io/antrea/pull/6734), [@Atish-iaf])
```

Comment:
```suggestion
  -  `antctl get bgproutes` to print the advertised BGP routes on the local Node. ([#6734](https://github.com/antrea-io/antrea/pull/6734), [@Atish-iaf])
```

Reply:
`antctl get bgproutes` is used to display the BGP routes advertised by the local Node.

Reply:
Update.